phpunit.xml

```diff
@@ -46,7 +46,4 @@
 			</exclude>
 		</whitelist>
 	</filter>
-	<listeners>
-		<listener class="SpeedTrapListener" file="tests/legacy/includes/listener-loader.php" />
```

Comment:
If we're no longer using this, perhaps we could remove this file as well?

Reply:
This file is still included in the code files inside `test/legacy`. So either we can remove that folder as well, or we need to keep the file.

Reply:
Can you show me where? Because when I did a search, the only place `listener-loader.php` shows up is the `listener-loader.php` file itself and in the `phpunit.xml` which you removed.

Reply:
Of course. WP's `listener-loader.php`  is referenced in Woo's `listener-loader.php` and that's it... me being supersmart as usual. The file is now removed.